### ISSUE TITLE
Automatic container deploy to Heroku failing

### ISSUE THREAD

**Issue by adr29truck** [adr29truck/express_scraper#29]
The GH action used to deploy to Heroku is failing to build the container.

## Solution proposal
* Manually build the container
* Install the Heroku CLI
* Tag and push the container
* Release the new container

My thought is to try and do this in the current Docker Hub action, as it manages to build the container. And those we only need to add a tag to it install the CLI and push it.

**Comment by adr29truck** [comment]
The following error arises when trying to deploy 
```   Successfully built 80272adfedb5
  Successfully tagged registry.heroku.com/***/web:latest
  Docker container built.
Pushing container to heroku registry...
   ›   Warning: Our terms of service have changed: 
   ›   https://dashboard.heroku.com/terms-of-service
   ▸    No images to push
Error: Pushing docker container failed.
Error undefined
```

**Comment by adr29truck** [comment]
Should have been fixed in #50
Will await merge to main with deploy before closing.

Awaits: #51

**Comment by te4-adrian-almetun-smeds** [comment]
Resolved in #51
